Orabot/EventHandlers/CustomMessageHandlers/LinkParsingMessageHandlers/OpenRaResourceCenterMapLinkMessageHandler.cs

```diff
@@ -18,7 +18,12 @@ public override void Invoke(SocketUserMessage message)
 		{
 			foreach (var matchedLink in GetMatchedLinks(message.Content))
 			{
-				var number = matchedLink.Substring(matchedLink.LastIndexOf('/') + 1);
+				var numberStr = matchedLink.Substring(matchedLink.LastIndexOf('/') + 1);
+				if (!int.TryParse(numberStr, out var number))
+				{
+					continue;
+				}
```

Comment:
This is technically against https://github.com/OpenRA/OpenRA/wiki/Coding-Standard if that applies here at all.